source/scene.js

```diff
@@ -146,7 +148,11 @@ function calcGridBounds( grid ) {
 this.executeBlock = function ( block, action ) {
     var blockName = block[ 0 ];
     var blockID = block[ 1 ];
-    this.blockExecuted( blockName, blockID );
+    
+    if(scenario.name !== "scenario_dummy"){
```

Comment:
There are some coding standard violations here. Nothing serious, just some missing spaces. I sent an email about it. It may seem trivial, but it does help with code cleanliness and readability.